changelogs/internal/newsfragments/1967.clarification

```diff
@@ -0,0 +1 @@
+Add `x-weight` property for sorting events rendered with the `event-group` shortcode.
```

Comment:
The custom property should be documented in `/openapi_extensions.md`.

Reply:
Thanks, I wasn't aware the custom keys are documented there. Have added `x-weight`.